tfc_systemd.cmake

```diff
@@ -10,10 +10,14 @@ function(tfc_systemd_service_file_in_component EXE_TARGET COMPONENT DESCRIPTION)
     FILES "${CMAKE_BINARY_DIR}/systemd/${EXE_TARGET}@.service"
     DESTINATION "${CMAKE_INSTALL_LIBDIR}/systemd/system/"
     COMPONENT ${COMPONENT}
-    CONFIGURATIONS Release
+    CONFIGURATIONS ${CMAKE_BUILD_TYPE}
   )
 endfunction()
 
 function(tfc_systemd_service_file EXE_TARGET DESCRIPTION)
-  tfc_systemd_service_file_in_component(${EXE_TARGET} ${EXE_TARGET} ${DESCRIPTION})
+  if (CMAKE_BUILD_TYPE STREQUAL "Release")
+    tfc_systemd_service_file_in_component(${EXE_TARGET} ${EXE_TARGET} ${DESCRIPTION})
+  else()
+    tfc_systemd_service_file_in_component(${EXE_TARGET} "${EXE_TARGET}-dbg" ${DESCRIPTION})
```

Comment:
væri ekki betra að hafa þetta bara -${CMAKE_BUILD_TYPE} ?

Reply:
nei þá myndi compnent-inn og debian skráin heita framework-release

nema að þú viljir það, ég hef í enga skoðun á því

Reply:
if else'ið má vera þarna, ég er bara að tala um að það geta verið fleiri build type en debug og release.

Reply:
ég skil ekki alveg breytinguna sem þú ert að leggja til

Reply:
```
    tfc_systemd_service_file_in_component(${EXE_TARGET} "${EXE_TARGET}-dbg" ${DESCRIPTION})
```
í 
```
    tfc_systemd_service_file_in_component(${EXE_TARGET} "${EXE_TARGET}-$CMAKE_BUILD_TYPE}" ${DESCRIPTION})
```